openapi-cli/src/test/resources/ballerina-to-openapi/response/status_code_errors_01.bal

```diff
@@ -0,0 +1,171 @@
+import ballerina/http;
```

Comment:
We can add license headers here right?

Reply:
We have not added license headers for the other test cases. We should add this. Will add the license header in a separate PR. Created an issue to track this: https://github.com/ballerina-platform/ballerina-library/issues/6043